README.md

```diff
@@ -1,39 +1,14 @@
-Pull Request Template
-=====================
+attn: Pull Request Template
```

Comment:
I asked about this when I joined and the rationale at the time was capitalizing our name increased its visibility on Facebook which increased traffic to the site. Lowercase letters look better in the logotype, however, so we use both.